web/models/email.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Askbywho.Email do
+  use Askbywho.Web, :model
+  alias Askbywho.Repo
+  alias Askbywho.Brand
+
+  schema "emails" do
```

Comment:
so at what point would an "email' be turned to a coherence user?  So if an user does a quick signup, shouldn't they just be a coherence user who hasn't created a password?  Thus, couldn't we just use the user table as the links to the brands created?  Then, we can try to convert users who don't have account information to add more detail?  Also, what does logging in buy us?  Maybe this is covered later in the code :)